airbyte-integrations/connectors/source-harvest/source_harvest/manifest.yaml

```diff
@@ -0,0 +1,2832 @@
+          field_path:
+            - task_assignments
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestPath
+        pagination_strategy:
+          type: CursorPagination
+          page_size: 1
+          cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+          stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+      partition_router: []
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updated_at
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%SZ"
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ config['start_date'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_time_option:
+        type: RequestOption
+        field_name: updated_since
+        inject_into: request_parameter
+  - type: DeclarativeStream
+    name: tasks
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api.harvestapp.com/v2/
+        path: /tasks
+        http_method: GET
+        request_parameters:
+          per_page: "50"
+        request_headers:
+          Harvest-Account-Id: "{{ config['account_id'] }}"
+        authenticator:
+          type: SelectiveAuthenticator
+          authenticator_selection_path: ["auth_type"]
+          authenticators:
+            api_token:
+              type: BearerAuthenticator
+              api_token: "{{ config['api_token'] }}"
+            oauth:
+              type: OAuthAuthenticator
+              client_id: "{{ config['client_id'] }}"
+              grant_type: refresh_token
+              client_secret: "{{ config['client_secret'] }}"
+              refresh_token: "{{ config['client_refresh_token'] }}"
+              refresh_request_body: {}
+              token_refresh_endpoint: https://id.getharvest.com/api/v2/oauth2/token
+        error_handler:
+          type: CompositeErrorHandler
+          error_handlers:
+            - type: DefaultErrorHandler
+              backoff_strategies:
+                - type: WaitTimeFromHeader
+                  header: Retry-After
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - tasks
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestPath
+        pagination_strategy:
+          type: CursorPagination
+          page_size: 1
+          cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+          stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+      partition_router: []
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updated_at
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%SZ"
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
This comment applies to all the `incremental_sync` that were previously implemented using `IncrementalHarvestStream`:
* Previous datetime format used was `%Y-%m-%dT%H:%M:%S%Z` instead of `%Y-%m-%dT%H:%M:%SZ` but I think it was an error so I'm comfortable with going with this change
* Without https://github.com/airbytehq/airbyte-internal-issues/issues/6314, the stream changes the value we take to update the state

Reply:
Agreed that the previous datetime format was likely an error.

Looks like the state based on the most recent record has been implemented, now, correct?